### ISSUE TITLE
Function Deprecated 

### ISSUE THREAD

**Issue by degrovejazz** [cwilso/volume-meter#17]
Just FYI. Many of the functions used here are or will be deprecated soon including navigator.getUserMedia and createScriptProcessor.

**Comment by Ajamuar** [comment]
Any alternative for these?

**Comment by cwilso** [comment]
Yes; getUserMedia has moved to MediaDevices, and scriptprocessors are replaced by Audio Worklets.  I haven't had time for a rewrite, though.

**Comment by aahedi** [comment]
Even last time it works well. I using it. Unfortunately, now not work.

**Comment by charleskyalo** [comment]
If you are looking for a newer modern example checkout [Volume meter Google Chrome labs ](https://googlechromelabs.github.io/web-audio-samples/audio-worklet/basic/volume-meter/) the source code is here [source code](https://github.com/GoogleChromeLabs/web-audio-samples/tree/main/src/audio-worklet/basic/volume-meter/)
it uses the `AudioWorkelt` interface.
